cmd/submit.go

```diff
@@ -240,6 +242,15 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	}
 	defer resp.Body.Close()
 
+	var jsonErrBody apiErrorMessage
```

Comment:
It probably makes sense to put this in the `if` block since we only need to do this work when there is an error response.

Reply:
👍